src/index.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+import _ from 'lodash';
+/* eslint-enable */
```

Comment:
- [ ] [OPTIONAL] You do not need to import `lodash` in this line. this will get rid of the ESlint errors you are facing.